open-metadata-conformance-suite/open-metadata-conformance-suite-server/src/main/java/org/odpi/openmetadata/conformance/tests/repository/instances/TestSupportedRelationshipPropertySearch.java

```diff
@@ -264,22 +264,22 @@ protected void run(TestPhase phase) throws Exception
                 RepositoryConformanceProfileRequirement.CONSISTENT_TYPES.getRequirementId());
 
         String end1TypeDefName = relationshipDef.getEndDef1().getEntityType().getName();
+        EntityDef knownEnd1EntityDef = (EntityDef) repositoryHelper.getTypeDefByName(workPad.getLocalServerUserId(), end1TypeDefName);
         EntityDef end1EntityDef = entityDefs.get(end1TypeDefName);
-        EntityDef knownEnd1EntityDef = (EntityDef) repositoryHelper.getTypeDefByName(workPad.getLocalServerUserId(), end1EntityDef.getName());
-        verifyCondition((end1EntityDef.equals(knownEnd1EntityDef)),
-                assertion0,
-                testTypeName + assertionMsg0,
-                RepositoryConformanceProfileRequirement.CONSISTENT_TYPES.getProfileId(),
-                RepositoryConformanceProfileRequirement.CONSISTENT_TYPES.getRequirementId());
+        verifyCondition( (end1EntityDef != null) && end1EntityDef.equals(knownEnd1EntityDef),
+                         assertion0,
+                         testTypeName + assertionMsg0,
+                         RepositoryConformanceProfileRequirement.CONSISTENT_TYPES.getProfileId(),
+                         RepositoryConformanceProfileRequirement.CONSISTENT_TYPES.getRequirementId());
 
         String end2TypeDefName = relationshipDef.getEndDef2().getEntityType().getName();
+        EntityDef knownEnd2EntityDef = (EntityDef) repositoryHelper.getTypeDefByName(workPad.getLocalServerUserId(), end2TypeDefName);
         EntityDef end2EntityDef = entityDefs.get(end2TypeDefName);
```

Comment:
As above -- if the connector does not implement supertypes, this will return `null` for many relationships and the assertion just below will always fail as a result.